### PR TITLE
Added a minimal product json example

### PR DESCRIPTION
**Pull request by lesniakania**

### CODE REVIEW

File: versionned/v1/v1-product_payload_minimal_sample.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"brand_name":"Awesome",
+	"sub_brand_name":"Awesome Casual",
+	"price_currency":"eur",
+  	"circular_design_strategy": ["material_cyclability"],
+	"variations":[
+		{
+			"product_description": "Very cool T-Shirt",
+      		"material_composition": "p1v1_material_composition",
+      		"assigned_colour_category": ["black"],
+      		"name":"Cool T-Shirt Black",
```
Comment (DfromM) [inline versionned/v1/v1-product_payload_minimal_sample.json:15]: can you fix the indentation? :)

File: versionned/v1/v1-product_payload_minimal_sample.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	"product_category":"clothing",
+	"size_country_code": ["eu"],
+	"brand_name":"Awesome",
+	"sub_brand_name":"Awesome Casual",
+	"price_currency":"eur",
+  	"circular_design_strategy": ["material_cyclability"],
+	"variations":[
+		{
+			"product_description": "Very cool T-Shirt",
+      		"material_composition": "p1v1_material_composition",
```
Comment (DfromM) [inline versionned/v1/v1-product_payload_minimal_sample.json:13]: maybe let's put something here like "100% Cotton" or "78% Viscose, 22% Polyamid" here to be clearer what this should be, it's supposed to resemble the material composition as on the textile label

**Review by DfromM — COMMENTED** [review]
I really like the idea of having a minimal json example here! In theory, some of the fields you added here are not mandatory, I guess that's intentional? The real minimal example would just include the product name I think. But I like this, I think it gives a good overview of what should be in there.

File: versionned/v1/v1-product_payload_minimal_sample.json (hunk 1 above)
Comment (lesniakania) [reply versionned/v1/v1-product_payload_minimal_sample.json]: not sure what happened here, fixed :)

File: versionned/v1/v1-product_payload_minimal_sample.json (hunk 2 above)
Comment (lesniakania) [reply versionned/v1/v1-product_payload_minimal_sample.json]: done :)

**Comment by lesniakania** [comment]
Hm, yeah, it was kind of minimal for all levels. Truly minimal would be:
```
```

**Pull request merged**